key_manager/management/commands/manage_openid_keys.py

```diff
@@ -0,0 +1,81 @@
+from datetime import timedelta
+
+from Cryptodome.PublicKey import RSA
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from oidc_provider.models import RSAKey
+
+from key_manager import settings
+from key_manager.models import ManagedRsaKey
+
+
+class Command(BaseCommand):
+    help = 'Manages default OpenID RSA key for Tunnistamo'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--list-before', action='store_true', help='List keys before management cycle')
+        parser.add_argument('--list-after', action='store_true', help='List keys after management cycle')
+        parser.add_argument('--list-only', action='store_true', help='Only list keys, do not manage')
+
+    def handle(self, *args, **options):
+        # Show key summary
+        if options['list_before'] or options['list_only']:
+            self.list_keys()
+        if options['list_only']:
+            return
+
+        valid_keys = False
+        # loop through all the keys
+        for rsakey in RSAKey.objects.all():
+            # check if key is managed, and if not, create it
+            managedkey, created = ManagedRsaKey.objects.get_or_create(
+                key_id=rsakey,
+                defaults={'created': timezone.now(), 'expired': timezone.now()})
```

Comment:
Instead of repeatedly calling `timezone.now()`, you might want to do `now = timezone.now()` before the loop.